.gitmodules

```diff
@@ -8,8 +8,9 @@
   path = external/nmodl
   url = https://github.com/BlueBrain/nmodl
 [submodule "external/Random123"]
-	path = external/Random123
-	url = https://github.com/BlueBrain/Random123.git
+  path = external/Random123
+  url = https://github.com/BlueBrain/Random123.git
 [submodule "CMake/hpc-coding-conventions"]
-	path = CMake/hpc-coding-conventions
-	url = https://github.com/BlueBrain/hpc-coding-conventions.git
+  path = CMake/hpc-coding-conventions
+  url = https://github.com/BlueBrain/hpc-coding-conventions.git
+  branch = master
```

Comment:
two issues here:
* specifying branch in submodule is not supported in all git versions
* we don't want to be on top of master all time? Imagine every cmake-format or clang-format changing the formatting every time hpc-conding-convetion is updated.

Reply:
Ok ;)